tabular/src/autogluon/tabular/features/feature_metadata.py

```diff
@@ -369,7 +371,19 @@ def print_feature_metadata_full(self, log_prefix='', print_only_one_special=Fals
             return output_str
 
     @classmethod
-    def from_df(cls, df):
+    def from_df(cls, df: pd.DataFrame):
+        """
+        Construct a feature metadata based on the inferred types of an input :class:`pd.DataFrame`.
```

Comment:
grammar, also might want to clarify what feature metadata object is & what it's used for

Reply:
fixed grammar. The description of FeatureMetadata is in the class docs of FeatureMetadata: http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-869/9/api/autogluon.task.html#featuremetadata